cds.test only once per file (#1507)
cap-js/docs: node.js/cds-test.md

cds.test = (...args) => (new Test).run(...args)
```

:::warning Run `cds.test` once per test file

> `@sap/cds` relies on server state like `cds.model`. If you run `cds.test` multiple times within the same test file, this can lead to a conflicting state and erratic behavior.

Reviewer: ```suggestion
`@sap/cds` relies on server state like `cds.model`. Running `cds.test` multiple times within the same test file can lead to a conflicting state and erratic behavior.
```

cds.test = (...args) => (new Test).run(...args)
```

> :::warning Run `cds.test` once per file!
> 
> `@sap/cds` relies on objects like `cds.model` being globally available. If you run `cds.test` multiple times within the same file can lead to a conflicting state and erratic behavior.
> 
> :::

Reviewer 2: ```suggestion
:::warning Run `cds.test` once per test file

`@sap/cds` relies on server state like `cds.model`. If you run `cds.test` multiple times within the same test file, this can lead to a conflicting state and erratic behavior.
:::
```
